content/en/blog/_posts/2022-03-25-kubelet-credential-provider.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: blog
+title: 'Kubernetes 1.24: Kubelet Credential Provider Reaches Beta'
+date: 2022-03-25
```

Comment:
```suggestion
date: 2022-05-09
```

Reply:
Definitely the 9th of May (a Monday)? I saw the 10th also mentioned.

Please update file paths to match the intended publication date.